text/0045-reserved-token-ids.md

```diff
@@ -0,0 +1,68 @@
+- Feature Name: `reserved_token_ids`
+- Start Date: 2022-07-14
+- MCIP PR: [mobilecoinfoundation/mcips#0045](https://github.com/mobilecoinfoundation/mcips/pull/45)
+- Tracking Issue: N/A
+
+# Summary
+[summary]: #summary
+
+MobileCoin 1.2 introduced the ability for the network to operate on different assets based on an unsigned 64-bit integer token ID. This MCIP exists primarily to maintain a canonical mapping of token IDs to underlying assets.
+
+# Motivation
+[motivation]: #motivation
+
+We will need to allocate token IDs for development and testing purposes on our various networks. We also wish to ensure a given token ID is unique for an underlying asset across our networks: development and test network(s) (e.g. AlphaNet, TestNet, etc.), and our main network (i.e. MainNet).
+
+It is also critically important that a given token ID not be re-used within a network, because (presumably) the new token is not a direct successor token to the previously existing token, and it is not considered reasonable to expect the original token to be completely burned via transaction to the verifiable burn address. If a holder of a particular token loses their private key, then the contents of their account cannot be verifiably burned.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+As a result, the primary purpose of this change is to ensure that the same token ID is not re-used on a given network, but secondarily to ensure that a given token ID is also not re-used *between* networks. 
+
```

Comment:
I would think that we would want to allow main net token ids on the test net in order to better simulate tokens on mainnet, prior to them being on mainnet.
Is the desire that one spins up their own local network for these kind of tests and sanity checks?